tests/test_pe_reports.py

```diff
@@ -104,3 +105,12 @@ def test_reports_bad_log_level():
         except SystemExit as sys_exit:
             return_code = sys_exit.code
         assert return_code == 1, "main() should exit with error"
+
+
+def test_get_orgs_from_db():
+    """Test the get_orgs function."""
+    query_result = [("12345", "Test Org 1", "TO_ID_1")]
+    with patch("psycopg2.connect") as mock_connect:
+        mock_connect.cursor.return_value.fetchall.return_value = query_result
+        result = pe_reports.data.db_query.get_orgs(mock_connect)
+    assert result == [("12345", "Test Org 1", "TO_ID_1")]
```

Comment:
First of all, I applaud you for adding a new test here! 👍   However, I'm not so sure that you are really testing your `get_orgs` query.  If I'm understanding this correctly (which I may not be), you are mocking up the DB connection and telling it what you want the results to be (`query_result`), then you are validating that the query results are the same as what you said you wanted the results to be- which they always should be.  Am I missing something or is the `get_orgs` query not really being tested here?

Reply:
Thanks @DJensen94 and @aloftus23 👍